.github/release.config.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+    branches: [
+        {
+            name: "main",
+            channel: "latest",
+            level: "minor",
+        },
+        {
+            name: "maintenance",
+            channel: "latest",
+            level: "patch",
+        },
+        {
+            name: "v1-lts",
+            channel: "zowe-v1-lts",
+            level: "patch",
+        },
+        // {
+        //     name: "next",
+        //     prerelease: true,
+        // }
+    ],
+    plugins: [
+        [
+            "@octorelease/changelog",
+            {
+                displayNames: {
+                    "zowe-explorer": "Zowe Explorer",
+                    "zowe-explorer-ftp-extension": "Zowe Explorer Extension for FTP",
+                    "zowe-explorer-api": "Zowe Explorer APIs",
+                },
+                headerLine: "## TBD Release",
+            },
+        ],
+        [
+            "@octorelease/lerna",
+            {
+                // Use Lerna only for versioning and publish packages independently
+                npmPublish: false,
+            },
+        ],
+        [
+            "@octorelease/npm",
+            {
+                $cwd: "packages/zowe-explorer-api",
+                aliasTags: {
+                    "latest": ["zowe-v2-lts"],
```

Comment:
Should this stay as `zowe-v2-lts` or should `latest` point to the latest v1 branch? Not familiar with how `aliasTags` is used here

Reply:
Good catch, I think it's fine here because it defines the "zowe-v2-lts" tag as an alias that should be published anytime the "latest" tag is published. On lines 14-15 of the release config, we define "zowe-v1-lts" as the tag for the "v1-lts" Git branch. So defining the "latest" <-> "zowe-v2-lts" alias should be fine because it doesn't affect publishes of the "zowe-v1-lts" tag.